Packs/SecureHealth/Integrations/SecureHealth/SecureHealth.py

```diff
@@ -0,0 +1,369 @@
+
+    # Get the last fetch time, if exists
+    # last_run is a dict with a single key, called last_fetch
+    last_fetch = last_run.get('last_fetch', None)
+    # Handle first fetch time
+    if last_fetch is None:
+        # if missing, use what provided via first_fetch_time
+        last_fetch = first_fetch_time
+    else:
+        # otherwise use the stored last fetch
+        last_fetch = int(last_fetch)
+
+    latest_created_time = cast(int, last_fetch)
+
+    incidents: List[Dict[str, Any]] = []
+
+    severity = ','.join(HELLOWORLD_SEVERITIES[HELLOWORLD_SEVERITIES.index(min_severity):])
+
+    # alerts = [
+    #     {"name": "test0", "severity": "Low", "ioc": "manygoodnews.com",
+    #     "devicelocation": '1200 University Avenue, Berkeley Office', "deviceuuid": 'ab33819fcc01'}
+    # ]
+    alerts = client.fetch(1)
+
+    for alert in alerts:
+        incident_created_time = 1601022903.826247  # (datetime.datetime.now() -datetime.datetime(1970,1,1)).total_seconds()
+        incident_created_time_ms = incident_created_time * 1000
+
+        js = {
+            "incidenttype": "DeviceDomainAccess",
+            "assetID": "bakedasset",
+            "ioc": {
+                "type": "domain",
+                "value": alert["ioc"]
+            }
+        }
+        incident = {
+            'name': alert["name"],
+            'assetid': js["assetID"],
+            'occurred': timestamp_to_datestring(incident_created_time_ms),
+            'rawJSON': json.dumps(js),
+            'severity': 4,
+            'CustomFields': {  # Map specific XSOAR Custom Fields
+                'deviceuuid': alert["deviceuuid"],
+                'devicelocation': alert["devicelocation"],
+                'ioc': alert["ioc"],
+                'assetid': alert["deviceuuid"]
+            }
+        }
+
+        incidents.append(incident)
+
+        if incident_created_time > latest_created_time:
+            latest_created_time = incident_created_time
+
+    next_run = {'last_fetch': latest_created_time}
+    return next_run, incidents
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    api_key = demisto.params().get('apikey')
+
+    # get the service API url
+    base_url = urljoin(demisto.params()['url'], '/api/v1')
+
+    # How much time before the first fetch to retrieve incidents
+    first_fetch_time = arg_to_timestamp(
+        arg=demisto.params().get('first_fetch', '3 days'),
+        arg_name='First fetch time',
+        required=True
+    )
+    assert isinstance(first_fetch_time, int)
+
+    proxy = demisto.params().get('proxy', False)
+
+    demisto.debug(f'Command being called is {demisto.command()}')
+    try:
+        headers = {
+            'Authorization': f'Bearer {api_key}'
+        }
+        client = None
+        try:
+            headers = {
+                'Authorization': f'Bearer 23498534098845934865984'
```

Comment:
i suggest removing header here

Reply:
This is a test token as it's a simulation of a PAN AppID type incident.  I was told by Pramukh to synthesize in absence of real hardware.

Reply:
BTW, can this be fixed up with a CLI pull and push?  Up till now everything was authorized through the Demisto web frontend

Reply:
Yes